android/modules/locale/src/java/ti/modules/titanium/locale/DateTimeFormatProxy.java

```diff
@@ -183,6 +183,7 @@ public void handleCreationDict(KrollDict properties)
 		this.resolvedOptions = new KrollDict();
 		this.resolvedOptions.putAll(options);
 		this.resolvedOptions.put(TiC.PROPERTY_LOCALE, locale.toString().replace("_", "-"));
+		this.resolvedOptions.put("timeZone", this.dateFormat.getTimeZone().getID());
```

Comment:
It seems like the locale should be a two-digit number. Also, maybe add the remaining properties now as well:

<img width="787" alt="Bildschirmfoto 2020-11-21 um 09 01 10" src="https://user-images.githubusercontent.com/10667698/99871095-20e8f380-2bd8-11eb-8280-879391677d59.png">

Reply:
Right. I know resolved options is still incomplete. This PR is only adding `timeZone` since it was easy to do.

Adding the rest of the missing properties may require a bit of a "hack". I would have to make a giant assumption about the Java `getDateInstance()`, `getTimeInstance()`, and `getDateTimeInstance()` methods returning a `SimpleDateFormat` type (which seems to be true) and parse its string pattern to determine what the rest of the properties should be. I'd prefer to avoid doing it this way, but I haven't found a better solution yet.

Reply:
Updated PR to add all of the other missing options.